pages/common/git-extras.md

```diff
@@ -0,0 +1,16 @@
+# git extras
+
+> Set of commands for `Git`.
```

Comment:
```suggestion
> Set of commands for `git`.
```

Reply:
See [my suggestion](https://github.com/tldr-pages/tldr/pull/6037#discussion_r637580012) below, which is more appropriate.